app/scripts/controllers/rif/rns/rnsjs-delegate.js

```diff
@@ -125,23 +136,59 @@ export default class RnsJsDelegate extends RnsDelegate {
    * @returns {Promise<>} TransactionReceipt of the latest transaction
    */
   createSubdomain (domainName, subdomain, ownerAddress, parentOwnerAddress) {
-    return this.rnsJs.subdomains.create(domainName, subdomain, ownerAddress, parentOwnerAddress);
+    const result = this.rnsJs.subdomains.create(domainName, subdomain, ownerAddress, parentOwnerAddress);
+    result.then(transactionReceipt => {
+      if (transactionReceipt) {
```

Comment:
Same as https://github.com/anarancio/nifty-wallet/pull/16/files#r421147676

Reply:
I reply on that PR.